File: src/gatling/scala/org/openapitools/client/model/OrderBusinessDocumentsDTO.scala

package org.openapitools.client.model


case class OrderBusinessDocumentsDTO (
    _upd: Option[DocumentDTO],
    _ukd: Option[DocumentDTO],
    _torgTwelve: Option[DocumentDTO],
    _sf: Option[DocumentDTO],
    _ksf: Option[DocumentDTO]
)
object OrderBusinessDocumentsDTO {
    def toStringBody(var_upd: Object, var_ukd: Object, var_torgTwelve: Object, var_sf: Object, var_ksf: Object) =
        s"""
        | {
        | "upd":$var_upd,"ukd":$var_ukd,"torgTwelve":$var_torgTwelve,"sf":$var_sf,"ksf":$var_ksf
        | }
        """.stripMargin
}

File: ympa_fsharp_giraffe_server/src/model/OrderBusinessDocumentsDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.DocumentDTO

module OrderBusinessDocumentsDTO =

  //#region OrderBusinessDocumentsDTO


  type OrderBusinessDocumentsDTO = {
    Upd : DocumentDTO;
    Ukd : DocumentDTO;
    TorgTwelve : DocumentDTO;
    Sf : DocumentDTO;
    Ksf : DocumentDTO;
  }
  //#endregion
